activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/services/common/security/jwt/JwtGrantedAuthorityConverter.java

```diff
@@ -34,8 +34,11 @@ public JwtGrantedAuthorityConverter(JwtAccessTokenProvider jwtAccessTokenProvide
     @Override
     public Collection<GrantedAuthority> convert(Jwt jwt) {
         Collection<GrantedAuthority> grantedAuthorities = new ArrayList<>();
-        for (String authority : jwtAccessTokenProvider.accessToken(jwt).getRoles()) {
-            grantedAuthorities.add(new SimpleGrantedAuthority(this.authorityPrefix + authority));
+        for (String role : jwtAccessTokenProvider.accessToken(jwt).getRoles()) {
+            grantedAuthorities.add(new SimpleGrantedAuthority(this.authorityPrefix + role));
+        }
+        for (String permission : jwtAccessTokenProvider.accessToken(jwt).getPermissions()) {
```

Comment:
should u use PERMISSION_?

Reply:
We can use PERMISSION_ but then we should override default behaviour of Spring which uses ROLE_ for both roles and authorities (there's no concept of permission in the AuthorizationManager).

Reply:
We can use PERMISSION_ but then we should override default behaviour of Spring which uses ROLE_ for both roles and authorities (there's no concept of permission in the AuthorizationManager).